Gemfile

```diff
@@ -0,0 +1,54 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.6.3'
```

Comment:
Dùng rails 2.7.1 nhé

Dùng nháy đôi luôn cho toàn bộ file ruby

Reply:
dạ